sys/kern/vm_map.c

```diff
@@ -80,6 +80,10 @@ vaddr_t vm_map_entry_end(vm_map_entry_t *ent) {
   return ent->end;
 }
 
+vm_map_entry_t *vm_map_entry_next(vm_map_entry_t *ent) {
+  return TAILQ_NEXT(ent, link);
+}
+
```

Comment:
I would add inline and replaced TAILQ_NEXT in line 233 and 325 vm_map.c with this function.